static/assets/app.js

```diff
@@ -293,6 +333,26 @@ app.controller("IndexController", function (UTILS, SETTING, VIEW, Room, Message,
     }
   };
 
+  /**
+   * Handle emojis
+   */
+  $scope.$on("whisper.Room:message", function (room, message) {
+    $timeout(function () {
```

Comment:
`$timeout` without time parameter is used to run function after template has rendered. It's an AngularJS thing.